svg/geometry/reftests/ellipse-003.svg

```diff
@@ -0,0 +1,17 @@
+<svg width="300" height="200"
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:html="http://www.w3.org/1999/xhtml">
+  <title>Ellipse coordinates and radii specified by percentage</title>
+  <html:link rel="help" href="https://svgwg.org/svg2-draft/geometry.html"/>
+  <html:link rel="match"  href="ellipse-ref.svg" />
+  <style>
+    ellipse {
+      cx: 25%;
+      cy: 60%;
+      rx: 20%;
```

Comment:
Maybe we should have a test with percentage and a scaled viewBox.

Reply:
And viewport units as well.

Reply:
These tests can be added in a subsequent PR.